db/1.8/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -825,6 +825,7 @@ Updates data in a database.
 
 This operation selects from a table at a regular interval and generates one message per obtained row.  Optionally, you can provide watermark and ID columns. If a watermark column is provided, the values taken from that column are used to filter the contents of the next poll, so that only rows with a greater watermark value are returned. If an ID column is provided, this component automatically verifies that the same row is not picked twice by concurrent polls.
 
+This operation does not support streaming, and indicates that the access to the results are done without any additional transformations to the payload. This behavior is identical to the the *Query Single* operation released on version 1.9.
```

Comment:
Do you have to say what version the Query Single operation is in?  If yes, change "released on" to "released in."